wandb/sdk/artifacts/artifact.py

```diff
@@ -1300,7 +1300,9 @@ def add_reference(
                 automatic integrity validation. Disabling checksumming will speed up
                 artifact creation but reference directories will not iterated through so the
                 objects in the directory will not be saved to the artifact. We recommend
-                adding reference objects in the case checksumming is false.
+                adding reference objects in the case checksumming is false. If
+                `checksum` is `False`, a new version will only be created if the reference URI
+                changes.
```

Comment:
question: Checking, but in case you already did -- do we happen to have an existing test for this?